griffon/services/core_queries.py

```diff
@@ -194,32 +194,28 @@ def execute(self, status=None) -> dict:
         return c["product_streams"]
 
 
-def async_retrieve_sources(self, purl, max_results=5000):
-    source_params = {
-        # "max_results": max_results,
+def async_retrieve_sources(self, purl):
+    params = {
+        "limit": 300,
```

Comment:
300 instead of default 10, so there is less requests overall?

Reply:
yes